src/Orleans.Core.Abstractions/IDs/SiloAddress.cs

```diff
@@ -177,7 +167,25 @@ public int GetConsistentHashCode()
 
             // Note that Port cannot be used because Port==0 matches any non-zero Port value for .Equals
             string siloAddressInfoToHash = Endpoint + Generation.ToString(CultureInfo.InvariantCulture);
-            hashCode = Utils.CalculateIdHash(siloAddressInfoToHash);
+            // TODO: the following is the equivalent to:
+            // hashCode = Utils.CalculateIdHash(siloAddressInfoToHash);
```

Comment:
don't just inline the code. At least create a separate method for it with the same name as the original, so that the method itself is a duplicate of what's in Utils.CalculateIdHash.
Otherwise it's very hard to keep them in sync